src/passes/OptimizeInstructions.cpp

```diff
@@ -986,46 +986,57 @@ struct OptimizeInstructions
   Expression* optimizeAddedConstants(Binary* binary) {
     uint32_t constant = 0;
     std::vector<Const*> constants;
-    std::function<void(Expression*, int)> seek = [&](Expression* curr,
-                                                     int mul) {
+
+    struct SeekState {
+      Expression* curr;
+      int mul;
```

Comment:
I guess this is pre-existing, but is 'mul' short for 'multiplier'? Could we maybe spend a few of our 80 columns and spell it out?

Reply:
oh, it makes more sense if you expand and read the function comment. either way.